### PR TITLE
Add "Liberty" metrics

### PR DESCRIPTION
**Pull request by jovergaag**
## Description of the change

We were trying to use our Supervision metrics to support the "Liberty to Prison" page, but with the change from `district` to `judicialDistrict` on that page it no longer made sense to share. So i created the over time metric and the snapshot metric here, along with the new filter type to support the new dimension. This also means that the `priorLengthOfIncarceration` dimension was removed from the supervision metrics and moved to the liberty metrics.

I also updated the ND district ids to match the new ids after we removed "OFFICE".

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes [#XXXX]

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally
- [ ] E2E have been run for Lantern changes ([Steps in the Readme](https://github.com/Recidiviz/pulse-dashboard#running-e2e-tests))

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/core/utils/filterOptions.ts [hunk 1]
```diff
@@ -548,25 +584,49 @@ export const NdPopulationFilterOptions: PopulationFilters = {
...
+      { label: "All", value: "ALL" },
+      { label: "South Central", value: "SOUTH CENTRAL" },
+      { label: "Northwest", value: "NORTHWEST" },
+      { label: "Northeast", value: "NORTHEAST" },
+      { label: "Southwest", value: "SOUTHWEST" },
+      { label: "Northeast Central", value: "NORTHEAST CENTRAL" },
+      { label: "East Central", value: "EAST CENTRAL" },
+      { label: "Unknown", value: "UNKNOWN" },
+      { label: "Southeast", value: "SOUTHEAST" },
+      { label: "North Central", value: "NORTH CENTRAL" },
```
Comment (Catacola) [inline src/core/utils/filterOptions.ts:629]: The values here should have underscores in place of spaces. e.g. `SOUTH_CENTRAL`

**Review by Catacola — CHANGES_REQUESTED** [review]

**Comment by jovergaag** [comment]
@Catacola bump :)

File: src/core/models/utils.ts [hunk 2]
```diff
@@ -259,6 +269,62 @@ export function createSupervisionPopulationTimeSeries(
...
+          race: record.race,
+          priorLengthOfIncarceration: record.prior_length_of_incarceration,
+        },
+        libertyDimensionDefaults,
+        enabledFilters
+      );
+    })
+    .sort((a, b) => {
+      return (a.year - b.year) * 12 + a.month - b.month;
+    });
```
Comment (Catacola) [inline src/core/models/utils.ts:298]: Is this still needed after your other fixes for the metric files today?

**Review by Catacola — APPROVED** [review]

File: src/core/models/utils.ts (hunk 2 above)
Comment (jovergaag) [reply src/core/models/utils.ts:298]: GOOD CATCH! I'll remove it.

**Pull request merged**